### PR TITLE
Improve the Query Log

### PR DESCRIPTION
**Pull request by roidelapluie**
Apply reviewers comments such as:

Capitalization
External links
Oxford comma

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: content/docs/guides/query-log.md [hunk 1]
```diff
@@ -6,16 +6,15 @@ sort_rank: 1
 # Using the Prometheus query log
 
 Prometheus has the ability to log all the queries run by the engine to a log
-file. This guide demonstrates how to use that log file, which fields it does
-contain and provides advanced tips about how to operate the log file.
+file. This guide demonstrates how to use that log file, which fields it
+contains, and provides advanced tips about how to operate the log file.
 
-The query log has been introduced in v2.16.0 and is disabled by default.
+The query log has been introduced in v2.16.0.
```
Comment (brian-brazil) [inline content/docs/guides/query-log.md]: I'd mention this in the first sentence, so 

to a log file, as of v2.16.0.

File: content/docs/guides/query-log.md [hunk 2]
```diff
@@ -25,42 +24,33 @@ To enable or disable the query log, two steps are needed:
...
-`/var/log/prometheus`. We will assume that Prometheus is running under a user
-called Prometheus:
-
-```shell
-# mkdir /var/log/prometheus
-# chown prometheus: /var/log/prometheus
-```
-
-Then, adapt the `prometheus.yml` configuration file:
+Firts, adapt the `prometheus.yml` configuration file:
```
Comment (brian-brazil) [inline content/docs/guides/query-log.md]: First

File: content/docs/guides/query-log.md [hunk 3]
```diff
@@ -25,42 +24,33 @@ To enable or disable the query log, two steps are needed:
...
-And, reload the Prometheus configuration:
+Then, [reload](../prometheus/latest/management_api/#reload) the Prometheus configuration:
+
 
 ```shell
 $ curl -X POST http://127.0.0.1:9090/-/reload
 ```
 
-Or, if Prometheus is not launched with `--web.enable-lifecycle`:
+Or, if Prometheus is not launched with `--web.enable-lifecycle`, you can trigger
```
Comment (brian-brazil) [inline content/docs/guides/query-log.md]: ...and you're not running on Windows

File: content/docs/guides/query-log.md [hunk 4]
```diff
@@ -112,17 +102,17 @@ present on any query:
...
 ```
 
-- params: the query itself. The start and end timestamp, the step and the actual
-  query.
-- stats: statistics. Currently, it contains internal engine timers.
-- ts: timestamp of the log line.
+- `params`: The query. The start and end timestamp, the step and the actual
+  query statement.
+- `stats`: Statistics. Currently, it contains internal engine timers.
+- `ts`: The timestamp of the log line, which is at the end of the query.
```
Comment (brian-brazil) [inline content/docs/guides/query-log.md]: The timestamp when the query ended.

**Comment by brian-brazil** [comment]
:+1:

**Comment by roidelapluie** [comment]
Note: I also removed the v in v2.16.0 as the v is an internal thing we use for the git tag, and the relase page says 2.x.x.

**Pull request merged**